locations/spiders/h_e_b_us.py

```diff
@@ -1,52 +1,20 @@
-import re
+from scrapy.spiders import SitemapSpider
 
-from scrapy import Spider
-from scrapy.http import JsonRequest
+from locations.structured_data_spider import StructuredDataSpider
 
-from locations.dict_parser import DictParser
-from locations.hours import DAYS_FULL, OpeningHours
 
-
-class HEBUSSpider(Spider):
+class HEBUSSpider(SitemapSpider, StructuredDataSpider):
     name = "h_e_b_us"
-    item_attributes = {"brand": "H-E-B", "brand_wikidata": "Q830621"}
+    HEB = {"brand": "H-E-B", "brand_wikidata": "Q830621"}
+    HEB_PLUS = {"brand": "H-E-B plus!", "brand_wikidata": "Q830621"}
     allowed_domains = ["www.heb.com"]
-    start_urls = ["https://www.heb.com/graphql"]
-    custom_settings = {"ROBOTSTXT_OBEY": False}
+    sitemap_urls = ["https://www.heb.com/sitemap/storeSitemap.xml"]
+    download_delay = 5
 
```

Comment:
Zyte may or may not help. I didn't want to just do it, because it might cost you money even if it fails.

```suggestion
    requires_proxy = True
```